addon/object/index.js

```diff
@@ -151,22 +153,24 @@ export const computed = decoratorWithParams(function(target, key, desc, params)
     throw new Error(`ES6 property getters/setters only need to be decorated once, '${key}' was decorated on both the getter and the setter`);
   }
 
-  let value;
-
   if (desc.writable === undefined) {
-    value = {
-      get: desc.get,
-      set: desc.set
-    };
+    const { get, set } = desc;
```

Comment:
I'll defer to @rwjblue on that one, don't really have a preference for style so long as it's consistent